person.rb

```diff
@@ -31,3 +31,6 @@ def of_age?
     @age.to_i >= 18
   end
 end
+
+# person = Person.new(22, 'maximilianus', parent_permission: false)
+# puts person.parent_permission
```

Comment:
- [ ] Please remove all commented code from your codebase to make your code clean and easy to maintain.